src/lib/ui/AccessCodeTable/AccessCodeTable.stories.tsx

```diff
@@ -1,6 +1,4 @@
-import MuiContainer from '@mui/material/Container'
-import Dialog from '@mui/material/Dialog'
-import DialogContent from '@mui/material/DialogContent'
+import { Container as MuiContainer, Dialog, DialogContent } from '@mui/material'
```

Comment:
@mikewuu Just want to note that we should always import like this from MUI everywhere (and for any libs, prefer named vs default if possible). That will be the friendliest for the bundler and treeshaking, and groups everything together under a single import.

Reply:
Yea! This fixed the weird not callable error. 

I usually prefer default imports though because I find named imports slowed down TS:

![CleanShot 2023-04-27 at 22 26 45](https://user-images.githubusercontent.com/11449462/234876415-a60c9841-7f59-40a1-917e-fb5145c600d5.png)

https://mui.com/material-ui/guides/minimizing-bundle-size/#development-environment